### ISSUE TITLE
Add driver_path option

### ISSUE THREAD

**Issue by y-ken** [shinji19/embulk-input-athena#1]
I have tried to test this cool plugin. But I have faced to this problem.
It needs introduce driver_path option as other jdbc driver has.

As workaround, I have installed jar which I got from AWS page and install that like following command in OSX.
`sudo cp AthenaJDBC41-1.1.0.jar /Library/Java/Extensions/`

```
java.lang.ClassNotFoundException: com.amazonaws.athena.jdbc.AthenaDriver
	at org.embulk.plugin.PluginClassLoader.loadClass(PluginClassLoader.java:257)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:264)
	at org.embulk.input.athena.AthenaInputPlugin.getAthenaConnection(AthenaInputPlugin.java:193)
	at org.embulk.input.athena.AthenaInputPlugin.run(AthenaInputPlugin.java:101)
	at org.embulk.exec.PreviewExecutor$2$1.run(PreviewExecutor.java:128)
	at org.embulk.spi.util.Filters$RecursiveControl.transaction(Filters.java:84)
	at org.embulk.spi.util.Filters.transaction(Filters.java:42)
	at org.embulk.exec.PreviewExecutor$2.run(PreviewExecutor.java:118)
	at org.embulk.input.athena.AthenaInputPlugin.resume(AthenaInputPlugin.java:82)
	at org.embulk.input.athena.AthenaInputPlugin.transaction(AthenaInputPlugin.java:77)
	at org.embulk.exec.PreviewExecutor.doPreview(PreviewExecutor.java:116)
	at org.embulk.exec.PreviewExecutor.doPreview(PreviewExecutor.java:104)
	at org.embulk.exec.PreviewExecutor.access$000(PreviewExecutor.java:30)
	at org.embulk.exec.PreviewExecutor$1.run(PreviewExecutor.java:74)
	at org.embulk.exec.PreviewExecutor$1.run(PreviewExecutor.java:71)
	at org.embulk.spi.Exec.doWith(Exec.java:22)
	at org.embulk.exec.PreviewExecutor.preview(PreviewExecutor.java:71)
	at org.embulk.EmbulkEmbed.preview(EmbulkEmbed.java:152)
	at org.embulk.EmbulkRunner.previewInternal(EmbulkRunner.java:215)
	at org.embulk.EmbulkRunner.preview(EmbulkRunner.java:107)
	at org.embulk.cli.EmbulkRun.runSubcommand(EmbulkRun.java:434)
	at org.embulk.cli.EmbulkRun.run(EmbulkRun.java:92)
	at org.embulk.cli.Main.main(Main.java:26)
```

**Comment by shinji19** [comment]
Thank you for testing and reporting!

I mistook build. Next build(v0.1.1) include athena jdbc. 
On the other hand, I challenge to introduce driver_path as other jdbc driver. Just a moment, please!

**Comment by shinji19** [comment]
Fix driver_path (v0.1.2)
https://github.com/shinji19/embulk-input-athena/tree/v0.1.2
